### ISSUE TITLE
Manager can't view users

### ISSUE THREAD

**Issue by sfrlaurent** [nilsteampassnet/TeamPass#2474]
### Steps to reproduce
1. connect as user , manager of a role
2.  list users 

### Expected behaviour
as it was in previous release (2.1.17.7) 
the manager lists all the users managed by the managers of the role

### Actual behaviour
the list is empty even if the manager creates a new user.

### Server configuration
**Operating system**:
ubuntu 16.04

Web server:
apache/2.4.18
Database:
mysql 14.14

PHP version:
php 7.0.30

Teampass version:
2.1.27.22
**Teampass configuration file:**

[tp.config.php.txt](https://github.com/nilsteampassnet/TeamPass/files/2580206/tp.config.php.txt)

**Updated from an older Teampass or fresh install:**
updated to 2.1.27.22 from 2.1.27.7

### Client configuration
chrome or firefox or IE

**Operating system:**
windows 7
#### Web server error log
no errors 

#### Log from the web-browser developer console (CTRL + SHIFT + i)
```
Insert the log here and especially the answer of the query that failed.
```

**Comment by nilsteampassnet** [comment]
I believe that you haven't adapted the manager delegation for the users.

PLease check https://teampass.readthedocs.io/en/latest/manage/manage-users/#add-a-new-user and especially `User is administrated by`.

Example with manager `U2` that has role `Test2` defined as bellow.
![image](https://user-images.githubusercontent.com/1197546/48509166-9afb7a80-e850-11e8-8af2-3c52be9ec2d5.png)

As `U2` is defined as Manager and associated to role `Test2`, he will be the manager of any other normal users that have the delegation to role `Manager of role Test2`. 
![image](https://user-images.githubusercontent.com/1197546/48509118-7b645200-e850-11e8-859a-1bbc958fa159.png)

As a consequence, the next users will be available from his list of users
![image](https://user-images.githubusercontent.com/1197546/48509273-ea41ab00-e850-11e8-9426-9910d8e41739.png)

I hope this clarifies the point.

**Comment by sfrlaurent** [comment]
Hello

I disagree that you closed the issue because the behavior you discribed is exactly the configuration I have.
here are some pictures:

the manager has this config :

[cid:image003.png@01D480B7.ACE07A30]

Some users config:

[cid:image002.png@01D480B8.068E8BD0]

this manager is unable to see the users of the role he is manager for.

[cid:image005.jpg@01D480B8.3ED08DE0]

Once again, the behavior was OK in the version 2.1.27.7 and the bug apear only since the upgrade to 2.1.27.22

Can reopen the issue ?
Cordialement,
Patrick LAURENT

SFR
DRSIO/DT/SI Réseau/Ingénierie Outils Opération

De : Nils Laumaillé [mailto:notifications@github.com]
Envoyé : mercredi 14 novembre 2018 21:06
À : nilsteampassnet/TeamPass <TeamPass@noreply.github.com>
Cc : LAURENT, Patrick <patrick.laurent@sfr.com>; Author <author@noreply.github.com>
Objet : Re: [nilsteampassnet/TeamPass] Manager can't view users (#2474)


Closed #2474<https://github.com/nilsteampassnet/TeamPass/issues/2474>.
